projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -1705,8 +1707,15 @@ public TestrigSettings getBaseTestrigSettings() {
       Path serializedVendorConfigPath, ConvertConfigurationAnswerElement answerElement) {
     Map<String, GenericConfigObject> vendorConfigurations =
         deserializeVendorConfigurations(serializedVendorConfigPath);
+    Map<String, GenericConfigObject> hostVendorConfigurations =
```

Comment:
This will interfere with resolution of the issue mentioned earlier to scan input directories recursively. Instead, we should have a new top-of-testrig-level folder